recipes/mailpit/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "mailpit" %}
+{% set version = "1.27.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/axllent/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 7c3e6a50727dc5936293ad43e66e368e26666fb58df222d39d9ca7be52562c42
```

```suggestion
{% set version = "1.27.11" %}

package:
  name: mailpit
  version: {{ version }}

source:
  url: https://github.com/axllent/mailpit/archive/v{{ version }}.tar.gz
  sha256: 7c3e6a50727dc5936293ad43e66e368e26666fb58df222d39d9ca7be52562c42
```
We don't need the `name` Jinja key here - the recipe is easier to read without it.